portality/app.py

```diff
@@ -145,32 +145,6 @@
         app.logger.debug(err)
 
 
-# Redirects from previous DOAJ app.
-# RJ: I have decided to put these here so that they can be managed 
-# alongside the DOAJ codebase.  I know they could also go into the
-# nginx config, but there is a chance that they will get lost or forgotten
-# some day, whereas this approach doesn't have that risk.
-@app.route("/doaj")
```

Comment:
I do support removing the legacy routes, but are you aware of how frequently they're used? (~200k in last 3 weeks)

https://doajkibana.cottagelabs.com/app/kibana#/discover?_a=(columns:!(context.request.url.full),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'apm-*',key:context.request.url.pathname,negate:!f,params:(query:%2Fdoaj,type:phrase),type:phrase,value:%2Fdoaj),query:(match:(context.request.url.pathname:(query:%2Fdoaj,type:phrase))))),index:'apm-*',interval:auto,query:(language:lucene,query:''),sort:!('@timestamp',desc))&_g=(refreshInterval:(pause:!t,value:0),time:(from:now-30d,mode:quick,to:now))

Reply:
Although you may notice some odd functions requested, like 'abstract'

Reply:
Ok, I think we should discuss this in the team meeting, and decide what to do.  I think maintaining them for 5 years is probably enough, but perhaps we can do something softer than just turn them off abruptly.  Or maybe this update is just not the right update in which to make that call, in which case we can revert the change.

Reply:
In fact, I'm just going to do that, and open an issue for removing legacy url support.